src/python/pants/option/global_options.py

```diff
@@ -201,3 +201,5 @@ def register_options(cls, register):
     register('--subproject-roots', type=list, advanced=True, fromfile=True, default=[],
              help='Paths that correspond with build roots for any subproject that this '
                   'project depends on.')
+    register('--force', type=bool, recursive=True,
```

Comment:
> Maybe, but a surprising lie since --cache-ignore would invalidate local fps and local results, and then ... hit the cache if enabled and use its results.

So, I actually think that that cuts directly to the root of this: my expectation of such an option (and probably a user's as well) is that not only would the target be "invalidated" (if they even know what that means), but _also_ that the cache not be used.

Particularly in the case of unit tests, if the goal is: "rerun this test, disabling all mechanisms that might prevent it from running" then both bits are necessary. And I think that invalidation (as a concept) is not something that users understand, but caching is.